unit-tests/py/ac.py

```diff
@@ -0,0 +1,31 @@
+import pyrealsense2 as rs
+from time import sleep
+
+status_list = []
+
+# Functions for using status list
+def status_list_empty():
+    global status_list
+    return not status_list
+
+def reset_status_list():
+    global status_list
+    status_list = []
+
+# Call back function for testing status sequences
+def list_status_cb( status ):
```

Comment:
Not `status_list_cb`? I prefer `status_list_callback` to make it more understandable